### ISSUE TITLE
non-monotonous timestamps when muxing certain AVC/H.264 sample

### ISSUE THREAD

**Issue by sneaker2** [gpac/gpac#864]
I have a sample taken from Blu-ray muxed to mp4 using mp4box. Playback in MPC-HC/LAV and ffplay is partly broken (it seems at least first and last 2(?) GOPs make screen grey and/or playback stop). When remuxing using ffmpeg -i "output.mp4" -c copy "remux.mp4" ffmpeg throws some errors:
```[mp4 @ 0000000002f0f020] Non-monotonous DTS in output stream 0:0; previous: -400
4, current: -4004; changing to -4003. This may result in incorrect timestamps in
 the output file.
[mp4 @ 0000000002f0f020] Non-monotonous DTS in output stream 0:0; previous: -400
3, current: -4004; changing to -4002. This may result in incorrect timestamps in
 the output file.
[mp4 @ 0000000002f0f020] Non-monotonous DTS in output stream 0:0; previous: -400
2, current: -4004; changing to -4001. This may result in incorrect timestamps in
 the output file.
[mp4 @ 0000000002f0f020] Non-monotonous DTS in output stream 0:0; previous: -400
1, current: -4004; changing to -4000. This may result in incorrect timestamps in
 the output file.
```

I have uploaded the sample to the MediaFire dropbox. To test I used a simple "_mp4box -add input.264 -output.mp4_". Version is latest nightly (gpac-0.7.2-DEV-rev175-ga99b084-master-win32.exe).

FWIW mkvtoolnix also had problems with this Blu-ray that now got fixed. Maybe their findings can give you a hint about what's going wrong:
[https://github.com/mbunkus/mkvtoolnix/issues/2028#issuecomment-312566525](https://github.com/mbunkus/mkvtoolnix/issues/2028#issuecomment-312566525)

**Comment by jeanlf** [comment]
Could you check if muxing using inband SPS/PPS does the trick: -add source.264:xps_inband ?

**Comment by sneaker2** [comment]
:xps_inband gets rid of the grey frames, i.e. all frames are shown without garbling but the stimestamp issue remains.

**Comment by jeanlf** [comment]
I believe this was fixed in GPAC 0.9.0 - reopen if needed
